### ISSUE TITLE
Indexing and querying

### ISSUE THREAD

**Issue by akiraplusplus** [ethereum/ethereum-org-website#2733]
We could use a developer resource around the topic of indexing & querying Ethereum blockchain data

**Comment by samajammin** [comment]
Agreed! I'll re-open https://github.com/ethereum/ethereum-org-website/issues/1564
